spec/lib/bamboozled/employee_spec.rb

```diff
@@ -112,5 +112,25 @@
       expect(location).to eq "https://api.bamboohr.com/api/gateway.php/alphasights/v1/employees/44259"
     end
   end
+
+  describe "#update" do
+    it 'updates an employee in BambooHR' do
+      xml = YAML.load_file('spec/fixtures/update_employee_xml.yml')
+      response = File.new('spec/fixtures/update_employee_response.json')
+      details = JSON.parse(File.read('spec/fixtures/update_employee_details.json'))
+
+      stub_request(:post, "https://x:x@api.bamboohr.com/api/gateway.php/x/v1/employees/1234").
+        with(xml).to_return(response)
+
+      employee = @client.employee.update(bamboo_id: '1234', employee_details: details)
+
+      employee["headers"].
+        must_equal({"content-type"=>["application/json; charset=utf-8"], "date"=>["Tue, 17 Jun 2014 19:25:35 UTC"]})
```

Comment:
Line is too long. [116/80]<br>Redundant curly braces around a hash parameter.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.